### PR TITLE
Adding warnings about thesaurus incomplete

### LINKED ISSUE #635: Update generate_template function and add UI for incomplete file
# Description

~~Add a `draft` key in the meta of a thesaurus to add a draft version of the thesaurus.~~

A new command `generate_missing_template` executes the command `generate_template` for each missing thesaurus.

~~Also, add the argument `draft` for the command `generate_template` and function `generate_language_template`.~~

And finally, an alert on "compare" and "reference" warns about the draft version of the thesaurus with a link to GitHub.

~~PS: To summarize the use of "draft" mode, this can be used to identify the incomplete thesaurus.~~

# Requirements

A PR will be available with these features.

# Additional Notes

image

### PR DESCRIPTION
**Pull request by ulyssear**
New PR for #636 
Will close #635

### CODE REVIEW

**Comment by geekygirlsarah** [comment]
Belated, but I think I like this. I think I have two main thoughts:

1. Can it just show one yellow box at the top rather than repeatedly throughout the page? It's not that pressing of an issue, and it's showing up in categories that are complete.
2. Can you take off "Warning!" as it's not like the values are wrong or anything, they're just missing.

**Review by geekygirlsarah — CHANGES_REQUESTED** [review]
See previous comment.

**Comment by ulyssear** [comment]
Hi ! Sorry for the time to answer, i'm busy this month.
To answer at your first question, i like your idea : it will be better to have a single "alert" at top of page instead at top of section indeed.
For your last question, of course i can edit that.
I don't know if i can do this the nearly days but i will be more available to do this the next week.

**Comment by ulyssear** [comment]
Hello!

I have made the modifications. Can you please check if all is good for the pull request?

There is a list of what it's done with the current pull request :
  - Each language for each category is checked (if there is a better solution to do that, I am interested): if there is at least one entry unknown, it's incomplete.
  - Alert for a category and a language (it's not a warning). The content is: "The category seems to be incomplete. Help us improve !" where "Help us improve!" is a link to the category (for the language).


Furthermore, i had a problem with docker when launching the project. A line has added in Dockerfile that fixed the problem. Can you please check that to see if it can be add to the stable branch ?

The problem was "Error: pg_config executable not found" fixed by adding the package via aptitude.

**Comment by ulyssear** [comment]
I removed the alerts inside the header of each category to have alerts at the top of page instead.

Screenshot of what it should render for a language's incomplete thesaurus below.

image

File: web/templates/compare.html
Comment (geekygirlsarah) [inline web/templates/compare.html:1]: Why is this file being added? Was this a file for testing?

**Comment by geekygirlsarah** [comment]
GitHub keeps telling me you made changes, but you keep force-pushing and so I can't compare and see what is new since the last review. Can you summarize the last 2 days of work since my last comment?

**Comment by ulyssear** [comment]
I created a new pull request, it will better for the merge. This pull request can be closed.
